hazelcast/src/main/java/com/hazelcast/client/impl/protocol/template/DynamicConfigTemplate.java

```diff
@@ -381,4 +381,20 @@ void addCacheConfig(String name, @Nullable String keyType, @Nullable String valu
                         @Nullable WanReplicationRef wanReplicationRef,
                         @Nullable HotRestartConfig hotRestartConfig);
 
+    /**
+     * Adds a new event journal configuration to a running cluster.
+     * If an event journal configuration for the same map or cache name already exists, then
+     * the new configuration is ignored and the existing one is preserved.
+     *
+     * @param mapName           name of {@code IMap} to use as event source
+     * @param cacheName         name of {@code ICache} to use as event source
+     * @param enabled           {@code true} to enable this event journal configuration, otherwise {@code false}
+     * @param capacity          capacity of event journal
+     * @param timeToLiveSeconds time to live (in seconds). This is the time the event journal retains items before removing them
+     *                          from the journal.
+     */
+    @Request(id = 17, retryable = false, response = ResponseMessageConst.VOID)
+    void addEventJournalConfig(@Nullable String mapName, @Nullable String cacheName, boolean enabled, int capacity,
```

Comment:
Not sure if that's possible to somehow express in the client-protocol, however I added validations on [client side](https://github.com/hazelcast/hazelcast/pull/11019/files#diff-56ee08fe8237fc406d39992cc8d36732R344) as well as on [member side](https://github.com/hazelcast/hazelcast/pull/11019/files#diff-cbf772471e0644a76279e71bca85d689R47)